src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -1645,7 +1656,11 @@ private function dumpValue($value, $interpolate = true)
             if (!method_exists($class, $method)) {
                 throw new InvalidArgumentException(sprintf('Cannot create closure-proxy for service "%s": method "%s::%s" does not exist.', $reference, $class, $method));
             }
-            $r = $this->container->getReflectionClass($class)->getMethod($method);
+            $r = $this->container->getReflectionClass($class);
```

Comment:
Yes, but once again, I'm not sure if the reflection could be made for a reason X or Y. Otherwise, There is indeed only one place to change (what was done in the first version of this PR)